Source/Concurrency/RefinementInstrumentation.cs

```diff
@@ -173,14 +173,14 @@ public override List<Cmd> CreateAssertCmds()
       var skipOrTransitionRelationAssertCmd = CmdHelper.AssertCmd(
         tok,
         Expr.Or(Expr.Ident(pc), Expr.Or(OldEqualityExprForGlobals(), transitionRelation)),
-        "Transition invariant violated in initial state");
+        "Some yield-to-yield fragment modifies state in a way that does not match the refined atomic action");
```

Comment:
Did you consider "a" or "this" instead of "some" in the new error message?

Reply:
Another suggestion:

"Some yield-to-yield fragment modifies state without matching the refined atomic action"

Reply:
I actually had "this", but then I realized that it does not really make sense because the token points to the procedure declaration. I actually would not know which token to choose to identify a yield-to-yield fragment, since they are dynamically scoped.

I like your suggestion, and we can go with "a" if you prefer.

Reply:
ok. "a" is better.

Reply:
Done.